### ISSUE TITLE
Request for Angular Version Support in Swiper

### ISSUE THREAD

**Issue by HCenggel** [nolimits4web/swiper#7197]
### Clear and concise description of the problem

Dear Swiper Team,

I hope this message finds you well. I am an avid user of Swiper and have found it to be an essential component in my Angular project. The features and functionality provided by Swiper greatly contribute to the user experience, and I appreciate the effort your team puts into maintaining and improving the library.

However, I have encountered a challenge in integrating Swiper seamlessly with Angular due to the absence of official Angular support. As Angular is a widely used framework in the development community, having a dedicated Angular version of Swiper would be immensely beneficial for many developers.

I would like to kindly request the development of an official Angular version of Swiper, complete with Angular-specific documentation, bindings, and best practices. This addition would not only simplify the integration process for Angular developers but also enhance the overall accessibility of Swiper within the Angular ecosystem.

Thank you for your time and continued dedication to improving Swiper.

Best regards

### Suggested solution

none

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [X] I'm willing to open a PR

**Comment by tomsun28** [comment]
Request +1 💗

**Comment by anjared** [comment]
For others that stumble upon this in the future.  If you follow this guide here it works great.  I followed this using Swiper v11.0.6 and Angular 17.2 and it worked with zero issues.  

https://medium.com/@fbanyai/swiper-10-angular-16-uncovered-1721bf32dfd6

The only thing that tripped me up is that I forgot to add the directive to the exports array in my shared module file but that has nothing to do with the article above, just something that I wasted some time on getting it to work.
